package.json

```diff
@@ -14,47 +14,50 @@
     "test": "npm run lint && gulp generate-assets && jest"
   },
   "dependencies": {
-    "acorn": "^8.0.4",
-    "ansi-colors": "^4.0.0",
+    "acorn": "^7.1.1",
+    "ansi-colors": "^4.1.1",
     "basic-auth": "^2.0.0",
     "basic-auth-connect": "^1.0.0",
-    "browser-sync": "^2.26.13",
+    "body-parser": "^1.14.1",
+    "browser-sync": "^2.11.1",
     "client-sessions": "^0.8.0",
-    "connect-flash": "^0.1.1",
     "cookie-parser": "^1.4.5",
-    "cross-spawn": "^7.0.3",
-    "del": "^6.0.0",
+    "cross-spawn": "^7.0.2",
+    "del": "^5.1.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "express-session": "^1.17.1",
-    "express-writer": "^0.0.4",
+    "express-session": "^1.13.0",
+    "express-writer": "0.0.4",
     "fancy-log": "^1.3.3",
+    "govuk-elements-sass": "^3.1.3",
     "govuk-frontend": "^3.10.1",
-    "gulp": "^4.0.2",
+    "govuk_frontend_toolkit": "^7.5.0",
+    "govuk_template_jinja": "^0.24.1",
+    "gulp": "^4.0.0",
     "gulp-nodemon": "^2.5.0",
     "gulp-sass": "^4.0.1",
-    "gulp-sourcemaps": "^3.0.0",
-    "humanize-duration": "^3.24.0",
+    "gulp-sourcemaps": "^2.6.0",
+    "inquirer": "^7.1.0",
     "keypather": "^3.0.0",
-    "luxon": "^1.22.0",
-    "marked": "^1.2.5",
-    "notifications-node-client": "^5.0.2",
-    "number-to-words": "^1.2.4",
-    "nunjucks": "^3.2.2",
+    "marked": "^1.2.0",
+    "notifications-node-client": "^4.7.2",
+    "nunjucks": "^3.2.1",
     "portscanner": "^2.1.1",
-    "prompt": "^1.0.0",
-    "querystring": "^0.2.0",
     "require-dir": "^1.0.0",
-    "sync-request": "^6.1.0"
+    "sync-request": "^6.0.0",
+    "universal-analytics": "^0.4.16",
+    "uuid": "^7.0.3"
```

Comment:
Not sure we need to downgrade dependencies… `humanize-duration`; I think that this is still being used for a Nunjucks filter, and used for work history breaks. `inquirer` is something that was added with a more recent version of the kit, might be worth investigating what changed to require this.